upgrade_guide/README.md

```diff
@@ -3,6 +3,35 @@
 The purpose of this guide is to detail changes made by successive
 versions of the Java driver.
 
+### 3.5.0
+
+`DowngradingConsistencyRetryPolicy` is now deprecated, see [JAVA-1752]. 
+It will also be removed in the next major release of the driver (4.0.0), 
+see [JAVA-1376].
+
+The main motivation is the agreement that `DowngradingConsistencyRetryPolicy`'s 
+behavior should be the application's concern, not the driver's. APIs provided by 
+the driver should instead encourage idiomatic use of a distributed system like 
+Apache Cassandra, and a downgrading policy works against this. It suggests that 
+an anti-pattern such as "try to read at ALL, but fall back to QUORUM if that fails" 
```

Comment:
nit: `try to read at QUORUM, but fall back to ONE`, would be probably the most likely scenario so would be better to document that.